### PR TITLE
Translation string fixes [MAILPOET-4758]

### PR DESCRIPTION
**Pull request by JanJakes**
## Description

Just a couple of string fixes based on discussions in #mailpoet-translations.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

https://github.com/mailpoet/mailpoet-premium/pull/644

## Linked tickets

[MAILPOET-4758]

## After-merge notes

_N/A_

[MAILPOET-4758]: https://mailpoet.atlassian.net/browse/MAILPOET-4758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/lib/Config/PopulatorData/Templates/WelcomeBlank1Column.php [hunk 1]
```diff
@@ -123,7 +123,7 @@ private function getBody() {
...
+                    "text" => __("<h1 style=\"text-align: center;\"><strong>Hi, new subscriber!</strong></h1>\n<p>&nbsp;</p>\n<p>[subscriber:firstname | default:Subscriber],</p>\n<p>&nbsp;</p>\n<p>You recently joined our list and we'd like to give you a warm welcome!</p>\n<p>&nbsp;</p>\n<p>Want to get to know us better? Check out some of our most popular articles: </p>\n<ol>\n<li><a href=\"https://www.mailpoet.com/the-importance-of-focus-when-writing/\">The Importance of Focus When Writing</a></li>\n<li><a href=\"https://www.mailpoet.com/write-great-subject-line/\">How to Write a Great Subject Line</a></li>\n<li><a href=\"https://www.mailpoet.com/just-sit-write-advice-motivation-ernest-hemingway/\">Just Sit Down and Write &ndash; Advice on Motivation from Ernest Hemingway</a></li>\n</ol>", 'mailpoet'),
```
Comment (johnolek) [inline mailpoet/lib/Config/PopulatorData/Templates/WelcomeBlank1Column.php]: (this comment also applies to the same content in the 12 column file)

We should probably update these to be the new URLs (and sometimes different titles) under `/blog/` so we don't have to rely on 301 redirects, which aren't working for one of the links:

https://www.mailpoet.com/blog/the-importance-of-focus-when-writing/
5 Ways to Improve Your Focus When Writing

---

https://www.mailpoet.com/blog/write-advice-motivation/
Title is the same for this one

---

https://www.mailpoet.com/blog/writing-next-great-email-subject-line/
Writing Your Next Great Email Subject Line

I'm assuming this is the correct article. The original, `https://www.mailpoet.com/write-great-subject-line`, 301s to `https://www.mailpoet.com/blog/write-great-subject-line/`, which 301s to `https://www.mailpoet.com/?p=5289`, which 404s.

File: mailpoet/assets/js/src/automation/editor/components/header/document_actions.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ export function DocumentActions({ children }): JSX.Element {
                     as="h1"
                   >
                     <VisuallyHidden as="span">
-                      {__('Editing workflow: ', 'mailpoet')}
+                      {__('Editing workflow:', 'mailpoet')}
```
Comment (johnolek) [inline mailpoet/assets/js/src/automation/editor/components/header/document_actions.tsx]: Should we not be referring to it as an automation in user-facing text like this?

**Review by johnolek — CHANGES_REQUESTED** [review]
Looks good overall! Just a couple of minor things

File: mailpoet/lib/Config/PopulatorData/Templates/WelcomeBlank1Column.php (hunk 1 above)
Comment (JanJakes) [reply mailpoet/lib/Config/PopulatorData/Templates/WelcomeBlank1Column.php]: Good catch!

**Comment by JanJakes** [comment]
@johnolek I just rebased both this and premium. There were things merged to `trunk` today, so I had to recheck the strings.

**Review by johnolek — DISMISSED** [review]
Looks good to me!

**Review by websupporter — APPROVED** [review]
lgtm

**Pull request merged**